apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -159,7 +159,8 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
             for obj in init:
                 obj["graded"] = False
             data["correct_answers"][name] = union_drawing_items(init, ans)
-        else:
+        # We only want to define a correct answer in certain cases
```

Comment:
Nit, spelling out what these "certain cases" are? Tweak as needed.

```suggestion
        # Only save a correct answer if it contains any elements.
```